### DISCUSSION TITLE
Orphaned Containers

### DISCUSSION THREAD

**Discussion by FlashyCoderDude** [itzg/docker-minecraft-server#1374] [Q&A]
I am trying to run several servers for my family on large tower.  I have a docker-compose file for both servers.  However, when I start one server, then attempt to start the 2nd server, I get a warning about "orphaned" containers.  If I REMOVE the orphaned containers, it kills the first server and vise-versa.

Any ideas on how to prevent this from happening?  I have specifically named both containers different things, but this is still of no help.

**Comment by itzg** [comment]
If you have both servers in the one compose file, which is what I normally do, then you only bring up the composition once. It sounds like you're running duplicate commands in the same directory and they're stepping on each other.

**Reply by FlashyCoderDude** [reply]
Ah, yes - I have them running on 2 separate files because I wanted to control each one independently.  If I define them both in the same file, would I be able to control them separately?

**Reply by itzg** [reply]
Yes, you can qualify commands like `up` with the service(s). 

If you want to always treat them completely separate, then placing each file a directory each is easiest. There's also a project argument you can pass to compose, but gets tedious and error prone.
